www/gatsby-node.js

```diff
@@ -116,6 +140,31 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           }
         })
 
+        // Create package readme
+        result.data.allNpmPackage.edges.forEach(edge => {
+          const slug = `/packages/${edge.node.title}/`
+
+          if (localPackagesArr.includes(edge.node.title)){
+            createPage({
+              path: slug,
+              component: slash(packageTemplate),
```

Comment:
instead of an if statement at top, could do:

`component: localPackagesArr.includes(edge.node.title) ? slash(packageTemplate) :  slash(remotePackageTemplate)`